plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/EntriesTable.java

```diff
@@ -59,8 +61,10 @@
 import static io.trino.spi.type.VarbinaryType.VARBINARY;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static java.util.Objects.requireNonNull;
+import static org.apache.iceberg.MetadataTableType.ALL_ENTRIES;
 import static org.apache.iceberg.MetadataTableType.ENTRIES;
 
+// https://iceberg.apache.org/docs/latest/spark-queries/#all-entries
```

Comment:
I didn't get why the same table would be needed under a different name.
Could you please elaborate on the need for it ?

Reply:
`$entries` returns the entries of the **_current_** snapshot. `$all_entries` returns entries for **_all_** snapshots. 
`$all_entries` covers `$entries`, but `$entries` allows inspecting the current status without filtering.